cdap-common/bin/service

```diff
@@ -72,6 +72,10 @@ done
 # Load user-modifiable configuration
 test -f "${CDAP_CONF}"/cdap-env.sh && source "${CDAP_CONF}"/cdap-env.sh
 
+# Set JAVA_HEAPMAX from variable defined in JAVA_HEAP_VAR
+JAVA_HEAPMAX=${!JAVA_HEAP_VAR}
```

Comment:
can we check if JAVA_HEAPMAX is already set first?  If any existing user is already setting JAVA_HEAPMAX in cdap-env.sh, this will break them on upgrade.